### PR TITLE
Refine Slurm CRD blueprint and rename deployment_name in test

### PR DESCRIPTION
**Pull request by nick-stroud**
This change:
- use consistent deployment names
- use n2d instead of n2 to improve obtainability
- use larger machine for compute partition
- increase dynamic count of partitions since there is no cost for dynamic nodes

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/examples/hpc-slurm-chromedesktop.yaml
Comment (harshthakkar01) [inline community/examples/hpc-slurm-chromedesktop.yaml:82]: Just for my understanding, why do we want to set this to large number ?

File: tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml
Comment (harshthakkar01) [inline tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml:20]: nit: would be good to change this to `slurm-crd{{ build[0:5] }}` for consistency.

**Comment by harshthakkar01** [comment]
Also,
1) Can you add label (release-chore) ? to make sure PR Label verification passes.
2) Run integration test locally to make sure, it is working fine.

Thanks.

File: community/examples/hpc-slurm-chromedesktop.yaml
Comment (nick-stroud) [reply community/examples/hpc-slurm-chromedesktop.yaml:82]: This is the max number of nodes that can be dynamically scaled to. If set to one then we can only create 1 node which is not much of a cluster. In a real world situation we would probably have a need to run multiple nodes. Since this is a max and nodes are dynamic, it will only deploy the nodes if there are jobs waiting for them.

File: tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml
Comment (nick-stroud) [reply tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml:20]: This was intentional, slurm will only use a few characters from the name (like 6 or maybe 8) when naming resources so if we use `slurm-crd-{build}` the build id will not make it into instance names. It will be clear that the instances are for slurm because the will append other identifiers like "controller".

**Comment by nick-stroud** [comment]
Integration test is running, I will wait until it passes to merge.

**Review by harshthakkar01 — APPROVED** [review]

File: tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml
Comment (nick-stroud) [reply tools/cloud-build/daily-tests/tests/hpc-slurm-chromedesktop.yml]: Infact `slurm_cluster_name` must be a subset of the deployment name. Test caught it. I am rerunning with an update.

**Pull request merged**
